browser-test/src/trusted_intermediary.test.ts

```diff
@@ -387,6 +387,34 @@ test.describe('Trusted intermediaries', () => {
     await validateScreenshot(page, 'cannot-add-client-invalid-email')
   })
 
+  test('expect client can added and directly apply to programs', async () => {
```

Comment:
Thanks for merging the two similar tests.  Would you mind updating the test description back to "expect client can be added and directly apply to programs" since the test does more than just verify the screenshot?